src/status_im/contexts/wallet/events.cljs

```diff
@@ -84,8 +84,9 @@
 (rf/reg-event-fx :wallet/close-account-page
  (fn [{:keys [db]}]
    (let [just-completed-transaction? (get-in db [:wallet :ui :send :just-completed-transaction?])]
-     (when-not just-completed-transaction?
-       {:fx [[:dispatch [:wallet/clear-account-tab]]]}))))
+     {:db (update db :wallet dissoc :current-viewing-account-address)
```

Comment:
Remove the current viewing account from DB once we close the account page, some checks depend on this value when the send flow  starts from wallet home